articles/storage/common/storage-require-secure-transfer.md

```diff
@@ -15,7 +15,7 @@ The "Secure transfer required" option enhances the security of your storage acco
 
 When you use the Azure Files service, any connection without encryption fails when "Secure transfer required" is enabled. This includes scenarios that use SMB 2.1, SMB 3.0 without encryption, and some versions of the Linux SMB client. 
 
-By default, the "Secure transfer required" option is disabled.
+By default, the "Secure transfer required" option is enabled.
```

Comment:
Please use this:

By default, the "Secure transfer required" option is enabled when you create new storage account in Azure Portal.